app/views/devise/sessions/_mfa_enrolment.erb

```diff
@@ -1,29 +1,30 @@
-<h1 class="govuk-heading-xl"><%= t('mfa_enrolment.heading') %></h1>
-
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-one-half">
-
-      <%= f.label :totp_code, class: "govuk-label" %>
+  <div class="govuk-grid-column-two-thirds">
```

Comment:
this should have stayed `-full` so it's not squeezed but it's using the full width of the three-quarters defined above